### PR TITLE
Choose from multiple image attributes.

### PR DESCRIPTION
**Pull request by thocevar**

### CODE REVIEW

**Comment by ajdapretnar** [comment]
@ales-erjavec Could you please check whether this breaks anything and merge if not, since we need this for the demo project and I'd super appreciate it if we could merge it soon. :smiley: Tomaž promised it's short.

File: orangecontrib/imageanalytics/widgets/owimagenetembedding.py [hunk 1]
```diff
@@ -33,6 +35,7 @@ class OWImageNetEmbedding(widget.OWWidget):
                ("Missing Images", Table)]
     auto_commit = Setting(True)
     token = Setting("")
+    img_attr = ContextSetting(0)
```
Comment (ales-erjavec) [inline orangecontrib/imageanalytics/widgets/owimagenetembedding.py]: Context settings need a corresponding `settingsHandler` definition (in this case probably a `DomainContextHandler`), and later in the `set_data` an {close,open}Context call pair .

File: orangecontrib/imageanalytics/widgets/owimagenetembedding.py [hunk 2]
```diff
@@ -99,28 +107,33 @@ def set_data(self, data):
...
+        self.img_attr_changed()
 
     def token_name_changed(self):
         self.profiler.set_token(self.token)
         if self.profiler.token and self.data:  # token is valid
             self.commit()
         self.set_info()
 
+    def img_attr_changed(self):
+        self.file_att = self.atts[self.img_attr]
```
Comment (ales-erjavec) [inline orangecontrib/imageanalytics/widgets/owimagenetembedding.py]: This raises an index error when switching between inputs with different number of image attributes:
```
--------------------------------------------------------------------------------
IndexError                                    Traceback (most recent call last):
  File "/Users/aleserjavec/workspace/orange3/Orange/canvas/scheme/widgetsscheme.py", line 699, in process_signals_for_widget
    handler(*args)
  File "/Users/aleserjavec/workspace/orange3-imageanalytics/orangecontrib/imageanalytics/widgets/owimagenetembedding.py", line 122, in set_data
    self.img_attr_changed()
  File "/Users/aleserjavec/workspace/orange3-imageanalytics/orangecontrib/imageanalytics/widgets/owimagenetembedding.py", line 131, in img_attr_changed
    self.file_att = self.atts[self.img_attr]
IndexError: list index out of range
```

**Comment by ajdapretnar** [comment]
Big thanks!

**Pull request merged**